src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,562 @@
+              <i class="fas fa-pencil-alt" aria-hidden="true" />
+            {{/button}}
+          {{/inline-edit-action}}
+        {{/table-td}}
+        {{#> table-td table-td--action="true"}}
+          {{#> dropdown id=(concat table--id inline-edit--row "-dropdown-kebab") dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+          {{/dropdown}}
+        {{/table-td}}
+      {{/table-tr}}
+    {{/table-tbody}}
+  {{/table}}
+{{/inline-edit}}
+```
+
+## Documentation
+
+### Accessibility
+All accessibility requirements for inputs apply to elements within inline edit.
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `id` | `[labelling element]` | Provides a reference for toggle button and/or editable content. |
+| `aria-label="descriptive text"` | `.pf-c-inline-edit__toggle > button` | Provides an accessible description for toggle button. **Required** |
+| `aria-labelledby="[labelling element] [toggle button id]"` | `.pf-c-inline-edit__toggle > button` | Provides an accessible description for toggle button. **Required** |
+| `aria-label="descriptive text"` | `[radio button group]` | Provides an accessible description for radio groups. **Required** |
+| `role="radiogroup"` | `[radio button group]` | Provides an accessible role for radio buttons groups. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-inline-edit` | `<form>`, `<div>` | Initiates the inline edit component. **Required** |
+| `.pf-m-inline-editable` | `.pf-c-inline-edit`, `.pf-c-inline-edit [block level element]` | Modifies an inline edit region for editable state. |
+| `.pf-c-inline-edit__value` | `*` | Initiates an inline edit value. **Required** |
+| `.pf-c-inline-edit__input` | `*` | Initiates an inline edit input. **Required** |
+| `.pf-c-inline-edit__label` | `*` | Initiates an inline edit label. |
+| `.pf-c-inline-edit__action` | `*` | Initiates an inline edit action (visible when inline edit region is active). **Required** |
+| `.pf-m-icon-button` | `.pf-c-inline-edit__action` | Modifies the action button spacing. |
+| `.pf-m-valid` | `.pf-c-inline-edit__action` | Modifies the action button state. |
+| `.pf-m-value-group` | `.pf-c-inline-edit__group` | Modifies group for value group. |
+| `.pf-m-input-group` | `.pf-c-inline-edit__group` | Modifies group for input group. |
+| `.pf-m-action-group` | `.pf-c-inline-edit__group` | Modifies group for action group. |
+| `.pf-m-icon-group` | `.pf-c-inline-edit__group` | Modifies an action group item spacing. |
+| `.pf-m-column` | `.pf-c-inline-edit__group` | Modifies an action group flex direction. |
+| `.pf-m-footer` | `.pf-c-inline-edit__group` | Modifies an inline edit group margin-top. |
+| `.pf-m-enable` | `.pf-c-inline-edit__action` | Exposes an inline edit action by default. |
```

Comment:
I think `enable` may be too arbitrary, my first thought is what is being enabled? What about expose or visible